patches/gettext_gui_translator_py.patch

```diff
@@ -0,0 +1,11 @@
+--- Mnemosyne-2.10.1/mnemosyne/libmnemosyne/gui_translators/gettext_gui_translator.py	2022-12-02 14:00:58.000000000 +0100
++++ mnemosyne/libmnemosyne/gui_translators/gettext_gui_translator.py	2023-05-27 17:04:27.950235592 +0200
+@@ -17,7 +17,7 @@
+         if os.path.exists("mo"):
+             self.lang_path = "mo"
+         else:  # pragma: no cover
+-            self.lang_path = os.path.join(sys.exec_prefix, "share", "locale")
++            self.lang_path = os.path.join("/app", "share", "locale")
```

Comment:
you shouldn't need this patch. setting the prefix should be enough.

Reply:
I'm not a python dev, not this app's dev and I'm very new to flatpack but I tried using flatpak builder and it would not work with --prefix=/app
@linuxrider was helping me with the manifest files and created them including the module ones

Reply:
I think this one might be correct, though, as it probably defaults to /usr. This is a Python variable and not build tool variable, for a change.